### PR TITLE
Fix: #24. Explicitly support legacy parsers.

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

- explicit client migration strategy

Fix: #24

### CODE REVIEW

File: draft-ietf-httpapi-ratelimit-headers.md [hunk 1]
```diff
@@ -326,6 +326,14 @@ http features such as {{?HPACK=RFC7541}}.
 
 A client MUST process the received `RateLimit` fields.
 
+A client SHOULD use a Structured Fields parser, but MAY reuse an existing parser,
```
Comment (ioggstream) [inline draft-ietf-httpapi-ratelimit-headers.md:329]: From https://github.com/httpwg/http-extensions/pull/1666/files#diff-f54a4297373343b3f70dd039988d859c0937adb15139178db08171b79f414452R153

File: draft-ietf-httpapi-ratelimit-headers.md [hunk 2]
```diff
@@ -326,6 +326,14 @@ http features such as {{?HPACK=RFC7541}}.
...
+for example to migrate from legacy fields supporting only integer values.
+In doing so, they SHOULD:
```
Comment (unleashed) [inline draft-ietf-httpapi-ratelimit-headers.md:331]: Does this apply to the first sentence "use a Structured Fields parser", or the second one "reuse an existing parser", or both? My assumption is it applies to the second so maybe we can reword it to make it explicit/more obvious?

Comment (ioggstream) [reply draft-ietf-httpapi-ratelimit-headers.md:331]: @unleashed feel free to reword :)

**Review by mnot — CHANGES_REQUESTED** [review]
We ended up getting consensus NOT to do this, because the potential interoperability (and perhaps security) issues that would result from having multiple parsers overwhelms the benefits that you might gain in transition. Basically, if you do this, you lose most of the benefits of SF.

**Comment by ioggstream** [comment]
@mnot thanks for this feedback! So let's skip this PR and close accordingly the relevant issues.
